### DISCUSSION TITLE
[Workaround] GTK3 program does not know natural scrolling of touchpad

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [SpiralLinux/SpiralLinux-project#147] [General]
There may be several ways to workaround but some of them have side effect.

**Comment by CicadaSeventeen** [comment]
This comment is for workaround approachs. Please do not reply here if you are not writing one.
1. Removing `xserver-xorg-input-xxx` excepting `xserver-xorg-input-libinput` `xserver-xorg-input-all` and `xserver-xorg-input-wacom`. For most touchpad `xserver-xorg-input-libinput` is good enough.

**Comment by CicadaSeventeen** [comment]
see [here](https://github.com/SpiralLinux/SpiralLinux-project/issues/142)

**Reply by CicadaSeventeen** [reply]
I wait for some days but do not see your reply about this. @geckolinux, what is your opinions on these `xorg-xsever-input-*` packages?

**Reply by geckolinux** [reply]
This will be fixed in the next release:
https://github.com/SpiralLinux/SpiralLinux-project/issues/142
